go/flags/endtoend/vttablet.txt

```diff
@@ -182,6 +182,7 @@ Usage of vttablet:
       --dba_pool_size int                                                Size of the connection pool for dba connections (default 20)
       --degraded_threshold duration                                      replication lag after which a replica is considered degraded (default 30s)
       --disable_active_reparents                                         if set, do not allow active reparents. Use this to protect a cluster using external reparents.
+      --disable_replication_manager                                      Disable replication manager to prevent replication repairs.
```

Comment:
We should use dashes for new flags. That will avoid future work as we move all of them to dashes.

Reply:
Fixed